src/audio/dai-zephyr.c

```diff
@@ -243,6 +245,8 @@ static enum dma_cb_status
 dai_dma_cb(struct dai_data *dd, struct comp_dev *dev, uint32_t bytes,
 	   pcm_converter_func *converter)
 {
+	struct processing_module *mod = comp_mod(dev);
+	struct copier_data *cd = module_get_private_data(mod);
```

Comment:
This fails IPC3 compilation because they are unused. Move them under `#if CONFIG_IPC_MAJOR4`

Reply:
Thanks for pointing this out. Fixed.